docs/pages/identity-security/usage/alerts.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Identity Activity Center Alerts
+sidebar_label: Alerts
+description: Describes the pre-built security detections that trigger alerts in Teleport Identity Activity Center.
+tags:
+ - identity-security
+---
+
+Teleport Identity Security provides 50 pre-built security detections that automatically create alerts for suspicious identity-related activities across your infrastructure. These detections monitor events from Teleport and integrated services like AWS, GitHub, and Okta to identify potential security risks.
+
+Teleport Identity Security alerts help you detect and respond to security threats by monitoring for:
+- Unusual authentication patterns
+- Privilege escalations
+- Configuration changes that affect security
+- Account compromises
+- Policy violations
+
+All detections are pre-configured and currently cannot be modified. The severity levels (Critical, High, Medium, Low) are defined by the Identity Security team based on the potential impact of each detection.
+
+![Alerts](../../../img/access-graph/alerts.png)
+
+
+## Detection categories
+Each category below represents an integrated service where Teleport monitors for security events and suspicious activities pulled from various integration audit logs. The detections within each category are further organized by severity level to help you prioritize your security response efforts.
+
+### AWS detections
+
+Teleport Identity Security monitors your AWS infrastructure for critical security events:
```

Comment:
- I would end the sentence with a `.`, not a `:`
- Why do we only mention "critical security events" here, but then enumerate a bunch of other (non-critical) detections below?

Reply:
Clarified